linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/FunctionEmitter.scala

```diff
@@ -2629,6 +2663,22 @@ private[emitter] class FunctionEmitter(sjsGen: SJSGen) {
         case IdentityHashCode(expr) =>
           genCallHelper("systemIdentityHashCode", transformExprNoChar(expr))
 
+        case WrapAsThrowable(expr) =>
+          val newExpr = transformExprNoChar(expr).asInstanceOf[js.VarRef]
+          js.If(
+              genIsInstanceOfClass(newExpr, ThrowableClass),
+              newExpr,
+              genScalaClassNew(JavaScriptExceptionClass, AnyArgConstructorName, newExpr))
+
+        case UnwrapFromThrowable(expr) =>
+          val newExpr = transformExprNoChar(expr)
```

Comment:
Also cast this to `js.VarRef` to ensure it is one?

Reply:
Oops, yes. I forgot that one.